draft-js-plugins-editor/src/Editor/__test__/index.js

```diff
@@ -78,8 +78,8 @@ describe('Editor', () => {
           plugins={plugins}
         />
       );
-      expect(result.node.props.onChange).to.eq(changeSpy);
-      expect(result.node.props.editorState).to.eq(editorState);
+      expect(result.getElement().props.onChange).to.eq(changeSpy);
```

Comment:
why is the use of this api changing?

Reply:
Because I bumped enzyme and result.node should be replaced with result.getElement() in new version.